src/default-envrc

```diff
@@ -1,6 +1,9 @@
+#!/usr/bin/env bash
+# the shebang is ignored, but nice for editors
+
 if type -P lorri &>/dev/null; then
-  eval \"$(lorri direnv)\"
+  eval "$(lorri direnv)"
```

Comment:
I *think* these escapes need to be there? not sure

Reply:
no they break the envrc:
```
direnv: loading ~/src/petgraph/.envrc                                                                                                                                                        
./.envrc: eval:5: unexpected EOF while looking for matching ``'
./.envrc: eval:6: syntax error: unexpected end of file
```

Reply:
Strange, that hasn’t happened to me before.

However:

```bash
$ help eval
eval: eval [arg ...]
    Execute arguments as a shell command.

    Combine ARGs into a single string, use the result as input to the shell,
    and execute the resulting commands.
```

So I guess the args get concatenated anyway by eval.